src/panels/lovelace/cards/hui-tile-card.ts

```diff
@@ -333,6 +333,7 @@ export class HuiTileCard extends LitElement implements LovelaceCard {
             .actionHandler=${actionHandler()}
             role="button"
             tabindex="0"
+            aria-labelledby="info"
```

Comment:
@steverep Is it ok to point to the tile info which contains the name and the state? Or should we have only the name? Otherwise I will use `aria-label`.

Reply:
Yes, name and state would be even better.